### ISSUE TITLE
FreeBSD support

### ISSUE THREAD

**Issue by monocasual** [monocasual/giada#200]
Original pull request: https://github.com/monocasual/giada/pull/218

**Comment by yurivict** [comment]
Could you please merge the latest FreeBSD patch to the master?

**Comment by monocasual** [comment]
@yurivict this is planned for 0.16.1, coming very soon. Please be patient :wink:

**Comment by yurivict** [comment]
Thank you.

**Comment by monocasual** [comment]
@yurivict FreeBSD branch squashed and merged into master. Looks good?

**Comment by yurivict** [comment]
The build breaks:
```
error: use of undeclared identifier 'kernelMidi'; did you mean 'm::kernelMidi'?
        if (kernelMidi::hasAPI(RtMidi::UNIX_JACK))
            ^~~~~~~~~~
            m::kernelMidi
./src/core/kernelMidi.h:39:11: note: 'm::kernelMidi' declared here
namespace kernelMidi
          ^
src/gui/elems/config/tabAudio.cpp:98:6: error: use of undeclared identifier 'kernelAudio'; did you mean 'm::kernelAudio'?
        if (kernelAudio::hasAPI(RtAudio::UNIX_JACK))
            ^~~~~~~~~~~
            m::kernelAudio
```

**Comment by monocasual** [comment]
Should be fixed now (commit d8b7b01779540f1775a6525b84af10d4852c516e).

**Comment by yurivict** [comment]
Still:
```
src/gui/elems/config/tabAudio.cpp:486:3: error: use of undeclared identifier 'conf'; did you mean 'm::conf'?
                conf::soundSystem = G_SYS_API_JACK;
                ^~~~
                m::conf
./src/core/conf.h:38:11: note: 'm::conf' declared here
namespace conf
          ^
src/gui/elems/config/tabAudio.cpp:488:3: error: use of undeclared identifier 'conf'; did you mean 'm::conf'?
                conf::soundSystem = G_SYS_API_PULSE;
                ^~~~
                m::conf
./src/core/conf.h:38:11: note: 'm::conf' declared here
namespace conf
          ^
```

**Comment by monocasual** [comment]
Hopefully fixed in 5051bf92fd5f16c416114561c0ddbe0f2c5f99e9 ...

**Comment by yurivict** [comment]
It works now.

Thank you for fixing the issues!
